.github/workflows/ts-packages.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         run: cat packages/contracts/gas-report.txt
 
       - name: Run codechecks
+        if: ${{ github.repository == 'ethereum-optimism/optimism' }}
```

Comment:
Maybe this?
```suggest
        if: ${{ github.event.pull_request.base.repo.full_name == 'ethereum-optimism/optimism' }}
```

[Source](https://github.community/t/is-there-a-way-to-tell-if-a-pr-is-from-a-forked-repository/134186/7).